.vscode/extensions.json

```diff
@@ -1,6 +1,7 @@
 {
     "recommendations": [
         "dbaeumer.vscode-eslint",
-        "esbenp.prettier-vscode"
+        "esbenp.prettier-vscode",
+        "orta.vscode-jest"
```

Comment:
this adds the jest test explorer - which is how you run and debug tests without manually configuring launch.json